### PR TITLE
Release 3.0.2

### PR DESCRIPTION
**Pull request by jrbourbeau**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes https://github.com/conda-forge/bokeh-feedstock/pull/86

### CODE REVIEW

**Comment by jrbourbeau** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ requirements:
     - python >=3.8
     - jinja2 >=2.9
     - contourpy >=1
-    - numpy >=1.11.3
+    - numpy >=1.16
```
[the remark this answers is not in the document]
Comment (jrbourbeau) [reply recipe/meta.yaml:30]: Whoops, I was looking at the default branch which contains https://github.com/bokeh/bokeh/pull/12602. You're right that the `3.0.2` tag has `numpy >=1.11.3`. I'll revert this change

**Comment by jrbourbeau** [comment]
Good catch @jakirkham -- let me know if you have any additional comments

**Comment by bryevdv** [comment]
👍 yes he numpy min ver change will land in 3.1

File: recipe/meta.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jakirkham) [reply recipe/meta.yaml]: No worries. Thanks! 🙏

**Comment by jakirkham** [comment]
Thanks all! 🙏

**Comment by jrbourbeau** [comment]
Thanks @jakirkham!

**Pull request merged**